yt/python/yt/cli/yt_binary.py

```diff
@@ -89,7 +90,7 @@
     show-spec, execute, execute-batch, sky-share, show-default-config, transfer-account-resources, transfer-pool-resources,
     generate-timestamp
 Subtools:
-    clickhouse, spark, idm
+    clickhouse, spark, idm, jupyter
```

Comment:
I do not fully agree on the part of argumentation about being too long (in general, our CLI tends to be verbose rather than cryptic), but the public naming for CHYT is indeed CHYT, and not ClickHouse over YT. Will fix that.